recipes/r-knitcitations/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.0.8' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-knitcitations
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/knitcitations_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/knitcitations/knitcitations_{{ version }}.tar.gz
+  sha256: 3b366d6b7cd02998d451f820a7883e6c5c315391a5301112a80c8b725969081e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
```

Comment:
Not sure why the win build is failing, but I've seen similar problems with a few packages. You can disable it for now and we'll figure it out later.